test

```diff
@@ -0,0 +1,146 @@
+        font-size: 17px;
+      }
+
+      @media screen and (min-width: 600px) {
+        .topnav {
+          justify-content: center;
+        }
+
+        .topnav a {
+          float: left;
+          display: block;
+          color: #344563;
+          font-size: 15px;
+          line-height: 20px;
+        }
+
+        .topnav a:hover {
+          background-color: #ddd;
+          color: black;
+        }
+
+        .topnav a.welcome {
+          display: none;
+        }
+
+        .topnav .icon {
+          display: none;
+        }
+      }
+
+      @media screen and (max-width: 600px) {
+        .topnav {
+          justify-content: space-between;
+        }
+        .topnav a:not(:first-child) {
+          display: none;
+        }
+        .topnav a.icon {
+          float: right;
+          display: block;
+        }
+        .topnav a.welcome {
+          display: block;
+          font-size: 20px;
+          line-height: 28px;
+          color: #172b4d;
+        }
+      }
+
+      @media screen and (max-width: 600px) {
+        .topnav.responsive {
+          position: relative;
+          display: block;
+          height: 755px;
+        }
+        .topnav.responsive .icon {
+          position: absolute;
+          right: 0;
+          top: 0;
+        }
+        .topnav.responsive a {
+          float: none;
+          display: block;
+          text-align: left;
+        }
+        .topnav.responsive a.welcome {
+          display: none;
+        }
+
+        .topnav.responsive a.porfolio {
+          margin-top: 50px;
+        }
+        .topnav.responsive a.vf {
+          margin-top: 50px;
+        }
+        .topnav.responsive a.bg {
+          margin-top: 50px;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <div class="topnav" id="myTopnav">
+        <a href="#home" class="welcome">Welcome</a>
+        <a href="#works" id="porfolio" class="rightNav porfolio">Porfolio</a>
+        <a href="#aboutme" class="rightNav vf">About</a>
+        <a href="#contactme" class="rightNav bg">Contact</a>
+
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </div>
+    </header>
+
+    <div style="padding-left: 16px">
+      <h2>Responsive Topnav Example</h2>
+      <p>Resize the browser window to see how it works.</p>
+    </div>
+
+    <script>
+      const icon = document.querySelector(".icon");
+      //const rightNav = document.querySelector(".rightNav");
+      const x = document.getElementById("myTopnav");
+      icon.addEventListener("click", () => {
+        x.classList.toggle("responsive");
+      });
+      //rightNav.addEventListener("click", () => {
+      //  x.classList.remove("responsive");
+      //});
+
+      document.querySelectorAll(".rightNav").forEach((n) =>
+        n.addEventListener("click", () => {
+          x.classList.remove("responsive");
+        })
+      );
+    </script>
+  </body>
+</html>
```

Comment:
- [ ] Kindly remove the `test` file from your repository and please do not commit test files to your GitHub repository. It makes your repository ugly and does not add anything important to the main app. If you want to create files for testing you can have them on your local machine and you could use the `.gitignore` to not commit them.

Reply:
Fixed.